src/Types.js

```diff
@@ -31,7 +31,7 @@ class Types {
     }
 
     static Boolean() {
-        return new Types(value => (value == null ? null : Boolean(value)));
+        return new Types(value => (value === null ? null : Boolean(value)));
```

Comment:
Well, the default value can be null as well right ?

Reply:
I thought the deserializers wouldn't be called at all when the value is `null` but yes, since they're used to deserialize the default value they should take care of it 👍